docroot/themes/custom/uids_base/scss/global.scss

```diff
@@ -304,6 +305,10 @@ body:not(.page-node-type-article, .page-node-type-page).layout-builder-disabled
 
   .layout__container {
     margin-bottom: $desktop-width-gutter;
+
+    &div[class*="uids-component--"]:last-child {
```

Comment:
@cory-skeers A default margin is set on all the layout builder sections to create better design spacing by default.  I added this to remove the default bottom margin for the last layout container on each page if there is a background color/pattern.